docs/guides/typescript.md

```diff
@@ -66,7 +66,7 @@ require("@nomiclabs/hardhat-waffle");
 // This is a sample Hardhat task. To learn how to create your own go to
 // https://hardhat.org/guides/create-task.html
 task("accounts", "Prints the list of accounts", async () => {
-  const accounts = await ethers.getSigners();
+  const accounts = await hre.ethers.getSigners();
```

Comment:
I think it's valuable to show that if you were using a global hre field, you have to turn it into an explicit access. But now that I re-read this, it wasn't explicit either.

Reply:
I changed this for the same reasons described in https://github.com/nomiclabs/hardhat/pull/1554#discussion_r660831402.

Reply:
I updated this in https://github.com/nomiclabs/hardhat/pull/1554/commits/5f11182769677497a57031b790baea1939fc7bf9 in response to https://github.com/nomiclabs/hardhat/pull/1554#discussion_r661469038